core/src/main/java/org/bitcoinj/core/AddressV1Message.java

```diff
@@ -53,6 +53,17 @@ public void addAddress(PeerAddress address) {
         addresses.add(address);
     }
 
+    @Override
+    public int messageSize() {
+        if (addresses == null)
+            return 0;
+        int size = VarInt.sizeOf(addresses.size());
+        for (PeerAddress addr : addresses) {
+            size += addr.getMessageSize(1);
+        }
+        return size;
+    }
+
```

Comment:
```
         return (addresses != null)
                ?  VarInt.sizeOf(addresses.size()) +
                    addresses.stream()
                        .mapToInt(addr -> addr.getMessageSize(1))
                        .sum()
                : 0;
```

or simpler if we can get rid of the null check.

Reply:
Except for the terniary this is done.